CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 - Introduce new administrative channel management endpoints `/admin/v1/realms/{realmId}/channels`.
 - Introduce new administrative subscription management endpoints `/admin/v1/realms/{realmId}/subscriptions`.
 - Introduce new administrative endpoint for deleting auth tokens `DELETE /admin/v1/realms/{realmId}/tokens/{tokenId}`
+- Introduce new administrative endpoint for deleting users `DELETE /admin/v1/realms/{realmId}/users/{userId}`
```

Comment:
**issue:** The given changelog entry does not cover all new delete endpoints.
**suggestion:** Extend this list to cover all new delete endpoints.

```suggestion
- Introduce new administrative endpoint for deleting channels `DELETE /admin/v1/realms/{realmId}/channels/{channelId}`.
- Introduce new administrative endpoint for deleting subscriptions `DELETE /admin/v1/realms/{realmId}/subscriptions/{subscriptionsId}`.
- Introduce new administrative endpoint for deleting users `DELETE /admin/v1/realms/{realmId}/users/{userId}`.
```

Reply:
The changelog already mentioned introduction of administrative endpoints for channels and subsctriptions. :thinking: